docs/content/pipelines/paimon-pipeline.md

```diff
@@ -0,0 +1,209 @@
+# Paimon Pipeline Connector
+
+The Paimon Pipeline connector can be used as the *Data Sink* of the pipeline, and write data to [Paimon](https://paimon.apache.org). This document describes how to set up the Paimon Pipeline connector.
+
+## What can the connector do?
+* Create table automatically if not exist
+* Schema change synchronization
+* Data synchronization
+
+How to create Pipeline
+----------------
+
+The pipeline for reading data from MySQL and sink to Paimon can be defined as follows:
+
+```yaml
+source:
+   type: mysql
+   name: MySQL Source
+   hostname: 127.0.0.1
+   port: 3306
+   username: admin
+   password: pass
+   tables: adb.\.*, bdb.user_table_[0-9]+, [app|web].order_\.*
+   server-id: 5401-5404
+
+sink:
+  type: paimon
+  name: Paimon Sink
+  metastore: filesystem
+  warehouse: /path/warehouse
+
+pipeline:
+  name: MySQL to Paimon Pipeline
+  parallelism: 2
+```
+
+Pipeline Connector Options
+----------------
+<div class="highlight">
+<table class="colwidths-auto docutils">
+   <thead>
+      <tr>
+        <th class="text-left" style="width: 25%">Option</th>
+        <th class="text-left" style="width: 8%">Required</th>
+        <th class="text-left" style="width: 7%">Default</th>
+        <th class="text-left" style="width: 10%">Type</th>
+        <th class="text-left" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>type</td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what connector to use, here should be <code>'paimon'</code>.</td>
+    </tr>
+    <tr>
+      <td>name</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>The name of the sink.</td>
+    </tr>
+    <tr>
+      <td>catalog.properties.metastore</td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Metastore of paimon catalog, supports filesystem and hive.</td>
+    </tr>
```

Comment:
Though `metastore` field is marked required, it has default value `filesystem` so user don't have to specify it explicitly.
```suggestion
    <tr>
      <td>catalog.properties.metastore</td>
      <td>optional</td>
      <td style="word-wrap: break-word;"><code>"filesystem"</code></td>
      <td>String</td>
      <td>Metastore of paimon catalog, supports filesystem and hive.</td>
    </tr>
```